tests/unit/compute-tests.js

```diff
@@ -169,6 +171,23 @@ describe("Compute Ship", () => {
   });
 
   describe("Compute method using hull scoped object", () => {
+
```

Comment:
@timliuhull what other test cases should be considered?

Reply:
this is fine